ui/src/components/header.tsx

```diff
@@ -210,11 +210,15 @@ export const Header = () => {
             </Button>
           </DropdownMenuTrigger>
           <DropdownMenuContent align='end'>
+            <Link to='/about'>
+              <DropdownMenuItem>About</DropdownMenuItem>
+            </Link>
             {user.hasRole('superuser') && (
               <Link to='/admin'>
                 <DropdownMenuItem>Admin</DropdownMenuItem>
               </Link>
             )}
+            <hr />
```

Comment:
Replace the generic `hr` with [`DropdownMenuSeparator`](https://github.com/eclipse-apoapsis/ort-server/blob/2c90f97e9543e7cba6d37ffd98e481ad4a78cf18/ui/src/components/ui/dropdown-menu.tsx#L171-L181).

Reply:
Mind elaborating why / what the effect is? Isn't generic better?

Reply:
Sure! Generic in this UI component context means _base unstyled HTML component_, so no CSS or JS. By using the UI components instead of these we gain styles (the linked separator component has Tailwind classes `-mx-1 my-1 h-px bg-muted`, designed to fit in with other dropdown elements) and everything that the underlying [Radix UI component](https://www.radix-ui.com/primitives/docs/components/dropdown-menu) does. This is probably not that much in the case of a separator, but what it often does is provide keyboard navigation, make things more accessible and handle different edge-cases.

Does that make sense?

Reply:
Makes sense. Though why not https://ui.shadcn.com/docs/components/separator?

Reply:
[If we look at it](https://github.com/eclipse-apoapsis/ort-server/blob/2c90f97e9543e7cba6d37ffd98e481ad4a78cf18/ui/src/components/ui/separator.tsx#L28-L29) compared to the dropdown separator, we see that it is different color (`bg-muted` vs. `bg-border`) and it has no margins. That means that someone has designed the dropdown separator to fit in with other dropdown elements so it's easier to get a polished look.

There may be some differences with the underlying Radix components as well, but probably not huge in the case of a separator.

Reply:
Can you spot any differences? 🤓 

`DropdownMenuSeparator`: 
![image](https://github.com/eclipse-apoapsis/ort-server/assets/2370887/dc4d86ca-a9b3-45d0-ab94-6a8a708ce503)

`Separator`:
![image](https://github.com/eclipse-apoapsis/ort-server/assets/2370887/2497ae71-672e-490f-85df-52515bd6abfd)

`hr`:
![image](https://github.com/eclipse-apoapsis/ort-server/assets/2370887/48324613-cd4f-436c-aef9-c5c170fc8938)

Reply:
I can, though I probably wouldn't say to like `` DropdownMenuSeparator`   best 😉

Reply:
Using component libraries is IMO not really about what each individual would like the best, but consistency. But it's [completely editable](https://github.com/eclipse-apoapsis/ort-server/blob/main/ui/src/components/ui/dropdown-menu.tsx), feel free to make a PR. 🤓